packages/cli/src/index.ts

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env node
 
 // MUST import first to apply preset from args and set ssz hasher
-import "./applyPreset.js";
+import "./preInitialization.js";
```

Comment:
the prover has this file copy-pasted, we won't need to set uv threadpool there but might wanna rename to be consistent 

https://github.com/ChainSafe/lodestar/blob/b107d4bed5258993702ecae5cc1a7d51ff8f6080/packages/prover/src/cli/applyPreset.ts

Reply:
Updated.